### PR TITLE
feat(homepage): create Homepage with its various components, and set …

### PR DESCRIPTION
**Pull request by fketta**
…up blogdata

[Home] #5

# Description

Created the various Homepage components and displayed them. Set up blogdata to display carousel. Implemented translation and Jest testing for all components at the exception of the BlogsCarousel due to an issue with React Slick that i'm currently working on fixing.
## Type of change

Please select options that are relevant.

- New feature (non-breaking change which adds functionality)

# Checklist:

-   [X] My code follows the style guidelines of this project
-   [X] I have performed a self-review of my own code
-   [X] I have commented my code, particularly in hard-to-understand areas
-   [X] I have added tests that prove my fix is effective or that my feature works

**Files changed**:

**Tests:**
src/components/BookingButton/test/BookingButton.test.js
src/components/BookingButton/test/snapshots/BookingButton.test.js.snap
src/components/HomePage/test/Banner.test.js
src/components/HomePage/test/ConnectionSection.test.js
src/components/HomePage/test/PurchasingSection.test.js
src/components/HomePage/test/TherapistsInfoSection.test.js
src/components/HomePage/test/snapshots/Banner.test.js.snap
src/components/HomePage/test/snapshots/ConnectionSection.test.js.snap
src/components/HomePage/test/snapshots/PurchasingSection.test.js.snap
src/components/HomePage/test/snapshots/TherapistsInfoSection.test.js.snap
src/components/HomePage/test/matchMedia.mock.js
src/components/Navbar/test/Navbar.test.js
src/components/Navbar/test/snapshots/Navbar.test.js.snap

**Images:**
public/assets/blog1.jpg
public/assets/blog2.jpg
public/assets/blog3.jpg
public/assets/blog4.jpg
public/assets/blog5.jpg
public/assets/blog6.jpg
public/assets/indexBanner.jpg
public/assets/therapists.svg

**Components:**
src/components/BookingButton/BookingButton.jsx
src/components/HomePage/Banner.jsx
src/components/HomePage/BlogsCarousel.jsx
src/components/HomePage/ConnectionSection.jsx
src/components/HomePage/PurchasingSection.jsx
src/components/HomePage/TherapistsInfoSection.jsx
src/components/Navbar/Navbar.jsx

**Other:**
src/pages/index.jsx
src/layout/layout.jsx
src/styles/globals.css

**Screenshots**:
image
image
image

### CODE REVIEW

**Review by GhaniBahri — APPROVED** [review]
Great job, as expected

File: src/util/firebase.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import { initializeApp } from "firebase/app";
 import { getAuth } from "firebase/auth";
+import { getFirestore } from "firebase/firestore"; // will be used later
```
Comment (GhaniBahri) [inline src/util/firebase.js:3]: this line should be deleted, firesore is imported on line 8, this is created ESLint error

Comment (GhaniBahri) [reply src/util/firebase.js:3]: also run `npm run format` to pass prettier test

**Review by Zohir-kk — APPROVED** [review]
Good job fella, thank you for the beautiful work

File: src/util/firebase.js (hunk 1 above)
Comment (fketta) [reply src/util/firebase.js:3]: Fixed and resolved!

**Pull request merged**